helm_deploy/whereabouts-api/values.yaml

```diff
@@ -1,10 +1,13 @@
 ---
 # Values here are the same across all environments
 generic-service:
+  nameOverride: whereabouts-api
+
+  replicaCount: 4
 
   image:
     repository: quay.io/hmpps/whereabouts-api
-    tag: latest
+    tag: app_version    # override at deployment time
```

Comment:
This is standard convention, see [hmpps-template-kotlin/values.yaml](https://github.com/ministryofjustice/hmpps-template-kotlin/blob/08ee6f57bbf3ba6e464fe6a3d7946fb9666f959a/helm_deploy/hmpps-template-kotlin/values.yaml#L9)